src/mono/wasm/runtime-test.js

```diff
@@ -57,10 +58,10 @@ if (is_browser) {
 
 	// We expect to be run by tests/runtime/run.js which passes in the arguments using http parameters
 	var url = new URL (decodeURI (window.location));
-	arguments = [];
+	var raw_args = [];
```

Comment:
Please define this above the closure instead of below, it's less confusing that way

Reply:
Agreed. Thanks for pointing this out.